### PR TITLE
GH #12 - Added Contribution Guidelines in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by hnarasaki**
GH #12 - Added Contribution Guidelines in CONTRIBUTING.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+4. Open a [pull request](https://help.github.com/en/articles/about-pull-requests) for the maintainers of the repository to review.
+5. Be sure to add the maintainers to the pull request as reviewers to ensure a speedy review.
+6. Upon reviewing the changes, your change may be:
+    * merged, requiring no further changes on your part.
+    * annotated with a set of changes that must be made before the change will be merged.
+
+## Unit Tests
+
+All changes submitted must include updated unit tests (excluding documentation and configuration changes).
+In order for your change to be accepted, the [TravisCI](https://travis-ci.com/indeedeng/cluster-preset) pipeline must complete successfully.
```
Comment (DuaneOBrien) [inline CONTRIBUTING.md:19]: Looking through the existing repo it doesn't look like there are many unit tests for the other parts of the code, and I don't think adding test here will kick off the travis pipeline. We could probably keep the unit tests requirement but drop the travis pipeline sentence.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+## Code of Conduct
+
+ClusterPreset is governed by the [Contributor Covenant v1.4.1](CODE_OF_CONDUCT.md).
```
Comment (DuaneOBrien) [inline CONTRIBUTING.md:23]: Could just changes this to say "This project" rather than ClusterPreset

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Contributing
+
+All contributions are welcome to this project!
+
+The general process for contributing code is as follows:
+
+1. Open an issue describing the desired feature, bug fix, or improvement you'd like to see before making any changes.
+2. Assign the ticket to yourself, and/or leave a comment on the issue that you would like to work on it.
```
Comment (DuaneOBrien) [inline CONTRIBUTING.md:8]: The contributor should wait for confirmation from the project before starting work, as this one isn't rigorously maintained.

**Review by DuaneOBrien — CHANGES_REQUESTED** [review]

**Comment by mmorrisontx** [comment]
Closing all pulls, as this repository is now archived.

**Pull request closed without merging**